packages/core/src/command/consolidated/get-snapshot-local.ts

```diff
@@ -125,3 +133,25 @@ async function postSnapshotToApi({
     }
   );
 }
+
+export function removeContainedPatients(bundle: Bundle): Bundle {
+  if (!bundle.entry) return bundle;
+
+  const updatedEntry = bundle.entry.map(entry => {
+    if (entry.resource && "contained" in entry.resource) {
```

Comment:
nit: could put `entry.resource` in a const and use that, so we don't reuse `entry.resource` everywhere